inventory/pkg/client/client.go

```diff
@@ -39,6 +39,7 @@ var zlog = logging.GetLogger("InfraAPIClient")
 type ResourceTenantIDCarrier = inv_v1.FindResourcesResponse_ResourceTenantIDCarrier
 
 const (
+	MaxMessageSize          = 40 * 1024 * 1024 // 40 MB
```

Comment:
shall we set different send/rcv max sizes? I would expect to be always an asymmetric config - do we need clients to send 40MB of messages?

Reply:
Done, actually we need only to change the recv size of the client. Server side, the default cfg is max Int already.